index.html

```diff
@@ -7,26 +7,59 @@
     <link rel="stylesheet" href="./styles/styles.css" />
   </head>
   <body>
-    <div class="container">
+    <div class="container-advantage">
       <h1 class="title">What do we offer</h1>
+      <hr class="highlight" />
       <div class="row advantage">
-          <div class="col advantage-col">
-          <img class="advantage-img" src="./images/support.png" alt="support">
+        <div class="col advantage-col">
+          <img class="advantage-img" src="./images/support.png" alt="support" />
```

Comment:
Your layout is limiting the styles you need. Look at the design - everything is there.